docs/concepts/spsp.md

```diff
@@ -0,0 +1,21 @@
+## SPSP
+
+[SPSP](https://github.com/interledgerjs/ilp-protocol-spsp) (Simple Payment Setup Protocol) is an 
+[application layer protocol](https://github.com/interledger/rfcs/blob/master/0001-interledger-architecture/0001-interledger-architecture.md#protocol-layers), 
+which uses HTTPS endpoints to securely exchange payment details, and then allows the user to send and receive value 
+over Interledger. 
+
+SPSP provides lightweight client/server APIs to send and receive value. An SPSP client sends (pay) value, and an 
+SPSP server receives value. If *pull* value is supported, a server can send value and a client can receive (pull) value. 
+
+When you start an SPSP server, it:
+* Connects to a `moneyd` instance
+* Creates a receiver
+* Opens a port to listen to
+* Generates a [payment pointer](https://github.com/interledger/rfcs/blob/master/0026-payment-pointers/0026-payment-pointers.md)<br/>
+A payment pointer is like an email address, but for sending/receiving value. 
+
+When you start an SPSP client, it:
+* Uses the recipient's payment pointer to directly obtain destination account and secret from the receiver
+* Connects with moneyd using the destination account and secret.
+* Sends value over the connection
```

Comment:
"Sends value over the connection (if it's in push payment mode) or receives value (if it's in pull payment mode)"

It's a little tricky to describe how SPSP works without describing STREAM, because SPSP is really just a light setup protocol for setting up a STREAM connection. I understand if that's out of scope for now though, I think this is a good way to describe what SPSP accomplishes without going too far into how it works.